src/LinAlg/hiopVector.hpp

```diff
@@ -209,6 +209,10 @@ class hiopVector
   virtual void copyFromDev() = 0;
   virtual void copyToDev() const = 0;
   virtual void copyFromDev() const = 0;
+  
+  /// @brief get number of negative/zero values according to the given tolerance 'tol'
+  virtual long long numOfNegVal_w_tol(const double &tol) const = 0;
```

Comment:
I think these should conform to the naming conventions we established in the developer readme - perhaps one of the following?

- `numOfNegVal`
- `numOfNegValWithTolerance`
- `countNegativeValues` (I think this is the clearest)

Although this does match the convention used in a couple other internal HiOp functions like `addConstant_w_patternSelect` - @pelesh @cnpetra how should we apply the style guide to these functions?

Reply:
how about 

`numElemsAbsLessThan` `numElemsLessThen`

Reply:
I vote for `numOfElemsAbsLessThan` and `numOfElemsLessThan`.